src/library/TxStats.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * @notice The following is a sdk implementation of the tx stats calculation.
+ * This code is not used in any test. To enable a test call, uncomment the
+ * necessary lines in src/index.ts and add a test case.
+ */
+
+import { getTxStats } from "../utils/stats";
+
+export class TxStats {
+
+  private options;
+  constructor(options: any) {
+    this.options = options;
+  }
+
+  /**
+  * Fetch user transactions statistics
+  * @param {number} startTimestamp Start timestamp of the input
+  * @param {number} endTimestamp End timestamp of the input
+  * @public
+  * @methods
+  */
+  getUserStats = async (startTimestamp: number, endTimestamp: number) => {
+    startTimestamp = 1623619086;
+    endTimestamp = 1623885486;
```

Comment:
`startTimestamp` `endTimestamp` are overwritten here, if the param bug is fixed this as well shouldve been updated